django/santropolFeast/member/views.py

```diff
@@ -886,15 +887,117 @@ def geolocateAddress(request):
     return JsonResponse({'latitude': lat, 'longtitude': long})
 
 
-def change_status(request, id):
-    if request.method == "POST":
-        client = get_object_or_404(Client, pk=id)
-        status = request.POST.get('status')
-        client.status = status
-        client.save()
+def clientStatusScheduler(request, pk):
```

Comment:
I would rather use **id** as argument name, as it is the client ID. **pk** is more a database term.